templates/lxc-archlinux.in

```diff
@@ -132,13 +138,23 @@ EOF
     [ ${nttys:-0} -gt 6 ] && echo \
       "You may want to modify container's /etc/securetty \
       file to allow root logins on tty7 and higher"
+    # fixes for systemd in lxc container
+    cat << EOF > ${path}/autodev
+#!/bin/bash
+cd ${LXC_ROOTFS_MOUNT}/dev
+mkdir net
+mknod net/tun c 10 200
+chmod 0666 net/tun
```

Comment:
What exactly is requiring net/tun?

All other templates don't set it up, instead leaving it up to the user to set it up with a simple:
lxc.mount.entry = /dev/net dev/net none bind,create=dir

Your approach is also going to fail inside an unprivileged container so I'm personally against it.